dockerfile_scripts/cray-libs.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -x
+
+# Install Cray libcxi. This requires grabbing the cassini/cxi headers
+# and installing them into ${HPC_DIR} so we can compile libcxi.
+cray_src_dir=/tmp/cray-libs
+mkdir -p $cray_src_dir && \
+    cd $cray_src_dir && \
+    git clone https://github.com/HewlettPackard/shs-cassini-headers.git && \
+    git clone https://github.com/HewlettPackard/shs-cxi-driver && \
```

Comment:
forgot the trainling ".git"

should be https://github.com/HewlettPackard/shs-cxi-driver.git

Reply:
if it closes without it, you can skip this.